osu.Game/Screens/Select/CarouselContainer.cs

```diff
@@ -184,6 +185,58 @@ public void SelectGroup(BeatmapGroup group, BeatmapPanel panel, bool animated =
             ScrollTo(selectedY, animated);
         }
 
+        public void Sort(FilterControl.SortMode mode) {
+            switch (mode) { 
+                case FilterControl.SortMode.Artist:
+                    groups.Sort((x, y) =>
+                    {
+                        return string.Compare(x.BeatmapSet.Metadata.Artist, y.BeatmapSet.Metadata.Artist);

```

Comment:
Can you replace these three lambda bodies by an expression ? (`groups.Sort((x, y) => string.Compare([...]);`)